pkgs/development/libraries/gobject-introspection/default.nix

```diff
@@ -54,14 +54,14 @@ stdenv.mkDerivation (finalAttrs: {
     # with LD_LIBRARY_PATH environment variable.
     (substituteAll {
       src = ./absolute_shlib_path.patch;
-      inherit nixStoreDir;
+      env = { inherit nixStoreDir; };
```

Comment:
You did a change in substitute-all.nix so this should be unnecessary now

Reply:
Thanks missed that one while rebasing!